config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -1259,7 +1259,6 @@ periodics:
           - --scenario=kubernetes_e2e
           - --
           - --deployment=node
-          - --gcp-project-type=node-e2e-project
```

Comment:
Why we need to do this only for the containerd performance tests, and not the other tests in containerd.?